libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -70,10 +70,14 @@ const ftx = {
 
 const swapProviders: Record<string, SwapProviderConfig> = {
   changelly: {
-    nameAndPubkey: Buffer.from(
-      "094368616e67656c6c790480d7c0d3a9183597395f58dda05999328da6f18fabd5cda0aff8e8e3fc633436a2dbf48ecb23d40df7c3c7d3e774b77b4b5df0e9f7e08cf1cdf2dba788eb085b",
-      "hex"
-    ),
+    nameAndPubkey: Buffer.concat([
+      Buffer.from([9]),
```

Comment:
Maybe worth adding a function to avoid the hardcoded integers ?

Reply:
No idea how we get "this number", for me has been "trial and error" method. Any suggestion?

Reply:
@sarneijim it's the name string length :)

Reply:
Done. Thanks!

Reply:
:+1: 
(by the way that's an example of possible regression to other providers if we got it wrong :wink: )